wiki/Beatmap_Editor/AiMod/de.md

```diff
@@ -0,0 +1,175 @@
+AiMod
+========
+
+**AiMod** ist ein Programm, welchen man im Beatmap Editor finden kann.
+Der **AiMod** kann über den Tastenkürzel **Strg + Shift + A** oder über **File**, dann **Open AiMod** geöffnet werden.
```

Comment:
change `**Strg + Shift + A**` to `` `Strg` + `Shift` + `A` ``
... I've asked this specific change 3 times already, is there a reason to not do this?